### PR TITLE
Add Babylon exporter AnimationGroup serializer.

### PR DESCRIPTION
**Pull request by Drigax**
Fix for .babylon scene serialization where animation groups aren't serialized. Added serializer for TargetAnimation and AnimationGroup.

### CODE REVIEW

File: src/Animations/animationGroup.ts [hunk 1]
```diff
@@ -21,6 +21,19 @@ export class TargetedAnimation {
      * Target to animate
      */
     public target: any;
+
+    public constructor(animation: Animation, target: any){
```
Comment (deltakosh) [inline src/Animations/animationGroup.ts]: that is a major breaking change :)

File: src/Misc/decorators.ts [hunk 2]
```diff
@@ -247,6 +247,16 @@ export class SerializationHelper {
                 destination.animations.push(animation.serialize());
             }
         }
+
+        if (source.animationGroups)
```
Comment (deltakosh) [inline src/Misc/decorators.ts]: animation groups are only hosted by the scene. They cannot be on an object

File: src/Animations/animationGroup.ts (hunk 1 above)
Comment (Drigax) [reply src/Animations/animationGroup.ts]: Can you help me find a better way? I only added the constructor due to how we initialize TargetAnimation via:
https://github.com/BabylonJS/Babylon.js/pull/6333/files#diff-10c0fe63a65cb9102072d2577dc27c04L180

which doesn't seem to work now that I added serialize().

Comment (Drigax) [reply src/Animations/animationGroup.ts]: Ah, i think i just wanted to be fancy with field initialization, calling the default constructor should work, right?

File: src/Misc/decorators.ts (hunk 2 above)
Comment (Drigax) [reply src/Misc/decorators.ts]: I understand. I think i should just modify the sceneSerializer instead :)

**Comment by deltakosh** [comment]
Thanks :) I let the CI finish and I will merge

**Pull request merged**